reana/reana_benchmark/analyze.py

```diff
@@ -307,31 +295,35 @@ def _build_histogram_plot(
 
 def _build_total_time_histogram(
     df: pd.DataFrame, plot_parameters: Dict
-) -> (str, Figure):
+) -> Tuple[str, Figure]:
     title = plot_parameters["title"]
+    data = df[df["status"] == WorkflowStatus.finished]
+    series = data["runtime"] + data["pending_time"]
     return (
         "histogram_total_time",
-        _build_histogram_plot(
-            df["runtime"] + df["pending_time"], 10, "total_time", title
-        ),
+        _build_histogram_plot(series, 10, "total_time", title),
     )
 
 
-def _build_runtime_histogram(df: pd.DataFrame, plot_parameters: Dict) -> (str, Figure):
+def _build_runtime_histogram(
+    df: pd.DataFrame, plot_parameters: Dict
+) -> Tuple[str, Figure]:
     title = plot_parameters["title"]
+    data = df[df["status"] == WorkflowStatus.finished]["runtime"].dropna()
```

Comment:
Very minor: I like how you filtered `data` and defined `series` in separate lines in `_build_total_time_histogram()` function. I guess same approach could be used here as well (and also `_build_pending_time_histogram()`, WDYT?
Since `_build_histogram_plot()` expects `series` parameter, it would be nicer to pass it instead of `data`

Reply:
Agree. Nice catch.

Reply:
Done